### PR TITLE
Updated ruby version to 2.3.3 and rails version to 4.2.6

### PR DESCRIPTION
**Pull request by beylarova**
Updated devise gem
Updated code style in models

### CODE REVIEW

File: config/secrets.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
```
Comment (avsej) [inline config/secrets.yml]: this project is public btw

Comment (melnikaite) [reply config/secrets.yml]: production secret is in env

Comment (avsej) [reply config/secrets.yml]: but the file itself says that secrets from this file have to be kept private.

File: config/application.rb [hunk 2]
```diff
@@ -43,15 +42,9 @@ class Application < Rails::Application
     # Configure the default encoding used in templates for Ruby 1.9.
     config.encoding = "utf-8"
 
-    # Configure sensitive parameters which will be filtered from the log file.
+    # Configure sensitive parameters which will be filtered from the log file.ф
```
Comment (avsej) [inline config/application.rb]: :+1:

**Pull request closed without merging**